src/app/activity/bounty/List.ts

```diff
@@ -28,145 +27,131 @@ export const listBounty = async (request: ListRequest): Promise<any> => {
-		dbRecords = bountyCollection.find({ 'claimedBy.discordId': listUser.user.id, status: { $ne: 'Deleted' }, 'customerId': request.guildId }).limit(DB_RECORD_LIMIT);
-		break;
-    case 'CLAIMED_BY_ME_AND_COMPLETE':
-        dbRecords = bountyCollection.find({ 'claimedBy.discordId': listUser.user.id, status: 'Completed', 'customerId': request.guildId }).limit(DB_RECORD_LIMIT);
-        break;
-	case 'DRAFTED_BY_ME':
-		dbRecords = bountyCollection.find({ 'createdBy.discordId': listUser.user.id, status: 'Draft', 'customerId': request.guildId }).limit(DB_RECORD_LIMIT);
-		break;
-	case 'OPEN':
-		dbRecords = bountyCollection.find({ status: BountyStatus.open, isIOU: { $ne: true }, 'customerId': request.guildId }).limit(DB_RECORD_LIMIT);
+		dbRecords = bountyCollection.find({ $or: [ { 'claimedBy.discordId': listUser.user.id }, { applicants: { $elemMatch: { discordId: listUser.user.id }}} ] , status: { $ne: 'Deleted' }, 'customerId': request.guildId }).sort({ status: -1, createdAt: -1 });
+		listTitle = "👷 Bounties Claimed or Applied For by Me";
+		openTitle = "Applied For"
 		break;
-	case 'IN_PROGRESS':
-		dbRecords = bountyCollection.find({ status: BountyStatus.in_progress, 'customerId': request.guildId }).limit(DB_RECORD_LIMIT);
-		break;
-	case 'PAID_BY_ME':
-		dbRecords = bountyCollection.find({ 'createdBy.discordId': listUser.user.id, status: BountyStatus.complete, isIOU: true, 'customerId': request.guildId }).limit(DB_RECORD_LIMIT);
-		IOUList = true;
-		break;
-	case 'UNPAID_BY_ME':
-		dbRecords = bountyCollection.find({ 'createdBy.discordId': listUser.user.id, status: BountyStatus.open, isIOU: true, 'customerId': request.guildId }).limit(DB_RECORD_LIMIT);
-		IOUList = true;
-		break;
-	}
-	if (!(await dbRecords.hasNext())) {
-		return await listUser.send({ content: 'We couldn\'t find any bounties!' });
+	default: 
+		dbRecords = bountyCollection.find({ $or: [ { status: BountyStatus.open } , { status: BountyStatus.in_progress }, { status: BountyStatus.in_review } ], isIOU: { $ne: true }, 'customerId': request.guildId }).sort({ status: -1, createdAt: -1 });
+		listTitle =  "💰 Active Bounties";
 	}
-	return await sendMultipleMessages(listUser, dbRecords, request.guildId, channelName, IOUList);
-};
 
-const sendMultipleMessages = async (listUser: GuildMember, dbRecords: Cursor, guildId: string, bountyChannelName: string, IOUList: boolean): Promise<any> => {
-	if (IOUList) {
-		while (await dbRecords.hasNext()) {
-			const record: Bounty = await dbRecords.next();
-			const messageOptions: MessageEmbedOptions = await generateListEmbedMessage(record, record.paidStatus, guildId);
-			if (record.paidStatus == PaidStatus.unpaid) {
-				messageOptions.footer = {
-					text: '💰 - paid | ❌ - delete ',
-				};
-			}
-			const message: Message = await (listUser.send( { embeds: [messageOptions] } ));
-			if (record.paidStatus == PaidStatus.unpaid) {
-				await message.react('💰');
-				await message.react('❌');
+	const listOfBounties: MessageEmbedOptions = {
```

Comment:
naming of `listOfBounties` relative to the embed options is a little confusing to me